apps/web/src/components/CopyToClipboard.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useRef, useState } from "react";
+import { CheckIcon } from "@heroicons/react/24/outline";
+
+import { useHover } from "~/hooks/useHover";
+import Copy from "~/icons/copy.svg";
+import { Tooltip } from "./Tooltip";
+
+export function CopyToClipboard({
+  label = "Copy to clipboard",
+  value,
+}: {
+  label?: string;
+  value: string;
```

Comment:
Could we create an interface for the props to keep consistency with other components?

Reply:
Done [bd3b3d6](https://github.com/Blobscan/blobscan/pull/478/commits/bd3b3d625dcb423a1b09e358e30f88d672159bbf)